src/layout/Header/ui/Header.tsx

```diff
@@ -1,25 +1,58 @@
-import { FC } from 'react';
+import { FC, useState } from 'react';
+
+import MenuIcon from '@mui/icons-material/Menu';
+import { Link } from 'react-router-dom';
 
 import { useTranslation } from 'shared/hooks/i18nHook';
+import { Button, Drawer } from 'shared/ui';
 
 import { LanguageMenu } from './LanguageMenu';
 
 export const Header: FC = () => {
     const { t } = useTranslation('common');
 
+    const [open, setOpen] = useState(false);
+
     return (
-        <header className="w-full h-20 flex justify-between items-center">
-            <div>Logo</div>
-            <nav>
-                <ul className="flex gap-4 items-center">
-                    <li>
-                        <a href="/">{t('login')}</a>
+        <header
+            style={{ boxShadow: '1px 2px 5px 0px rgba(0,0,0,0.07)' }}
```

Comment:
if you use inline styles try to create styles object outside component if it's possible to avoid creating a new object every time when component is re-rendered